src/templates/default/_gitlab-ci.yml

```
@@ -60,6 +60,9 @@ building:
     - building
   stage: deploying
   image: registry.gitlab.com/greenhousegroup/meh/images/k8s-helm:2
+	retry:
+		max: 1
+		when: script_failure
```

```suggestion
  retry:
    max: 1
    when: script_failure
```
Je gebruikt hier tab characters